### PR TITLE
feat(*): reuse shared from brandsdk

### PR DESCRIPTION
**Pull request by fulopdaniel**

### CODE REVIEW

File: packages/animation-curve-block/tailwind.config.js [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 module.exports = {
     presets: [require('@frontify/fondue/tailwind'), require('../shared/tailwind.config')],
-    content: ['src/**/*.{ts,tsx}', '../shared/src/**/*.{ts,tsx}'],
+    content: [
+        'src/**/*.{ts,tsx}',
+        '../shared/src/**/*.{ts,tsx}',
+        'node_modules/@frontify/guideline-blocks-settings/dist/**/*.es*.js',
```
Comment (ragi96) [inline packages/animation-curve-block/tailwind.config.js:8]: I think there should be a better solution than exporting from the `node_modules`

Comment (fulopdaniel) [reply packages/animation-curve-block/tailwind.config.js:8]: We do the same in web-app. 🤔

Comment (ragi96) [reply packages/animation-curve-block/tailwind.config.js:8]: Ok maybe we should open a task to solve it on both ends, it's just really akward

Comment (ragi96) [reply packages/animation-curve-block/tailwind.config.js:8]: We need to import `import '@frontify/guideline-blocks-settings/styles';` so it won't be needed anymore

File: examples/asset-upload/package.json [hunk 2]
```diff
@@ -15,25 +15,24 @@
...
-        "postcss": "8.4.28",
-        "prettier": "2.7.1",
-        "sinon": "15.0.2",
-        "tailwindcss": "3.2.7",
-        "typescript": "4.8.4"
+        "postcss": "8.4.29",
+        "prettier": "2.8.8",
+        "sinon": "15.2.0",
+        "tailwindcss": "3.3.3",
+        "typescript": "4.9.5"
```
Comment (SamuelAlev) [inline examples/asset-upload/package.json:30]: Can go to 5 ;)

File: packages/shared/package.json [hunk 3]
```diff
@@ -18,46 +18,34 @@
...
-        "@uiw/react-codemirror": "4.19.9",
-        "escape-html": "1.0.3",
-        "slate": "0.94.1",
-        "slate-react": "0.97.1",
-        "tinycolor2": "1.4.2"
+        "@frontify/app-bridge": "3.0.0-beta.88",
+        "@frontify/fondue": "12.0.0-beta.316",
+        "@frontify/guideline-blocks-settings": "0.29.3",
+        "@uiw/codemirror-extensions-langs": "4.21.12",
+        "@uiw/react-codemirror": "4.21.12"
```
Comment (SamuelAlev) [inline packages/shared/package.json:48]:
```suggestion
        "@frontify/app-bridge": "^3.0.0-beta.88",
        "@frontify/fondue": "12.0.0-beta.316",
        "@frontify/guideline-blocks-settings": "^0.29.3",
        "@uiw/codemirror-extensions-langs": "^4.21.12",
        "@uiw/react-codemirror": "^4.21.12"
```

File: examples/asset-upload/package.json [hunk 4]
```diff
@@ -15,25 +15,24 @@
...
     },
     "dependencies": {
-        "@frontify/app-bridge": "3.0.0-beta.67",
-        "@frontify/fondue": "12.0.0-beta.309",
-        "@frontify/fondue-tokens": "3.3.0-next.5",
-        "@frontify/guideline-blocks-settings": "0.27.0",
+        "@frontify/app-bridge": "3.0.0-beta.88",
```
Comment (SamuelAlev) [inline examples/asset-upload/package.json:33]:
```suggestion
        "@frontify/app-bridge": "^3.0.0-beta.90",
```
everywhere

File: examples/asset-upload/package.json [hunk 5]
```diff
@@ -15,25 +15,24 @@
...
+        "@frontify/fondue": "12.0.0-beta.316",
+        "@frontify/guideline-blocks-settings": "0.29.3",
```
Comment (SamuelAlev) [inline examples/asset-upload/package.json:35]:
```suggestion
        "@frontify/guideline-blocks-settings": "^0.29.3",
```
Everywhere

File: package.json [hunk 6]
```diff
@@ -24,24 +24,24 @@
...
-        "vitest": "0.25.0"
+        "vitest": "0.34.3"
     },
     "dependencies": {
-        "@frontify/fondue": "12.0.0-beta.309",
-        "glob": "10.0.0",
+        "@frontify/fondue": "12.0.0-beta.316",
+        "glob": "10.3.4",
         "react": "18.2.0",
         "react-dom": "18.2.0"
```
Comment (SamuelAlev) [inline package.json:46]: Could you change the deps (in all packages) to always be `^version` except for Fondue?

Comment (fulopdaniel) [reply package.json]: Added everywhere 👍

File: examples/asset-upload/package.json (hunk 4 above)
Comment (fulopdaniel) [reply examples/asset-upload/package.json]: Bumped

File: examples/asset-upload/package.json (hunk 2 above)
Comment (fulopdaniel) [reply examples/asset-upload/package.json]: Updated everywhere

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**